source/_integrations/swiss_public_transport.markdown

```diff
@@ -41,6 +41,21 @@ name:
 
 The public timetables are coming from [Swiss public transport](https://transport.opendata.ch/).
 
-### Defining a custom polling interval
+## Actions
+
+The Swiss public transport integration has the following action:
+
+- `swiss_public_transport.fetch_connections`
+
+### Action `swiss_public_transport.fetch_connections`
+
+Fetch the connections for a specific instance.
+
+| Data attribute | Optional | Description                                              |
+|------------------------|----------|----------------------------------------------------------|
+| `config_entry_id`      | No       | The ID of the Swiss public transport config entry to get data from.|
```

Comment:
is this the station ID, as listed on the stationboard?

Reply:
No, the ID of the `integration`, as it is for the `mealie` integration. For each connection (Zurich --> Geneva) an instance of the integration exists, and one has to choose one of those.

Reply:
I see. Could you maybe add an example to the description? I think it would make it easier to understand.

Reply:
Perfect, thank you!